src/bank.ts

```diff
@@ -11,7 +12,15 @@ export class Bank extends ServerNode {
     return await this.getPaginatedData("/accounts", options);
   }
 
-  // TODO: PATCH  /accounts/<account_number>
+  /**
+   * Updates the given server account's trust on the bank.
+   * @param accountNumber the account number of the server to update
+   * @param trust the trust of the the server
+   * @param serverAccount the account for the server node in which the account number is the node identifier and the signing key is the node identifier signing key
+   */
+  async updateAccount(accountNumber: string, trust: number, serverAccount: Account) {
+    return await this.patchData(`/accounts${accountNumber}`, serverAccount.createSignedMessage({ trust: trust }));
```

Comment:
- Please use the shorthand for `trust` instead of `trust: trust`
- The endpoint should have a slash between (change it to `/accounts/${accountNumber}`)